### PR TITLE
feat: add ssl expiry checker

### PR DESCRIPTION
**Pull request by harishv7**
Fixes ISOM-1051

### CODE REVIEW

File: check-ssl-expiry/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "check-ssl-expiry",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (kishore03109) [inline check-ssl-expiry/package.json:7]: Nit: can add a run script here

File: check-ssl-expiry/index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const https = require("https");
+
+// List of domains to check
+const domains = ["example.com", "google.com", "facebook.com"];
```
Comment (kishore03109) [inline check-ssl-expiry/index.js:4]: Ideally use a csv + csv file reader! Thinking we can hardcode the csv value from the redirections directly, then we can import it as is into our backend

**Review by kishore03109 — APPROVED** [review]
Approving with nits

File: check-ssl-expiry/index.js (hunk 2 above)
Comment (harishv7) [reply check-ssl-expiry/index.js]: good point, will add in future PRs

File: check-ssl-expiry/package.json (hunk 1 above)
Comment (harishv7) [reply check-ssl-expiry/package.json]: added

**Pull request merged**